src/pymovements/datasets/potec.py

```diff
@@ -149,10 +150,22 @@ class PoTeC(DatasetDefinition):
                     'md5': 'cffd45039757c3777e2fd130e5d8a2ad',
                 },
             ],
+            'precomputed_events': [
+                {
+                    'resource': 'd8pyg/',
```

Comment:
this won't work as mirrors was removed from the definition

Reply:
done